flink-cdc-cli/src/test/java/org/apache/flink/cdc/cli/parser/YamlPipelineDefinitionParserTest.java

```diff
@@ -426,7 +427,9 @@ void testParsingFullDefinitionFromString() throws Exception {
                     Collections.emptyList(),
                     Collections.emptyList(),
                     Collections.emptyList(),
-                    Configuration.fromMap(Collections.singletonMap("parallelism", "1")));
+                    Configuration.fromMap(
+                            Collections.singletonMap(
+                                    "local-time-zone", ZoneId.systemDefault().toString())));
```

Comment:
Seems it's not necessary to pass `local-time-zone` explicitly since it will be accessed when constructing `PipelineDef` anyway